App.js

```diff
@@ -26,29 +20,39 @@ const storeData = async (value) => {
 
 const getData = async () => {
 	  try {
-	const jsonValue = await AsyncStorage.getItem('@tasks')
-	 return jsonValue != null ? JSON.parse(jsonValue) : null;
-        } catch(err) {
-		console.log(err);
+      const jsonValue = await AsyncStorage.getItem('@tasks')
+      
+      console.log('inside get promise', jsonValue, JSON.parse(jsonValue))
+      return jsonValue != null ? JSON.parse(jsonValue) : null;
+    } catch(err) {
+      console.log(err);
```

Comment:
I moved the `storeData` and `getData` functions to the very top of the file to make sure we are able to call them later on. I left a `console.log` by accident, but I recommend checking your data every step of the way.

Reply:
ok thanks for your help , i really appreciate it